app/utils.ts

```diff
@@ -37,3 +38,34 @@ export function getContract(
 
   return new Contract(address, ABI, getProviderOrSigner(library, account) as any);
 }
+
+// thanks @ajsantander -- (https://github.com/ajsantander/ovm-og-multisigwallet/blob/master/scripts/validate-owners.js)
+export const validateOwners = async ({ owners, required }) => {
```

Comment:
Ed, how hard would it be to highlight the offending address(es)? Something like a red border around the address that caused the problem? If this isn't too difficult, I think it would be very useful in helping users actually resolve the problem once detected.

Reply:
great suggestion thanks @apbendi , implemented